axelrod/strategies/axelrod_first.py

```diff
@@ -310,15 +312,16 @@ def __init__(self):
         super().__init__()
 
     @staticmethod
-    def score_history(my_history, opponent_history, score_map):
+    def score_history(my_history: List[Action], opponent_history: List[Action], score_map: Dict[Tuple[Action, Action], int]) -> int:
```

Comment:
As for the imports, I guess I will have to make a change to all the files which have typehints.
Do you want me to tackle all of them, or just in any PRs from now?